__tests__/OmnichannelChatSDK.spec.ts

```diff
@@ -1507,10 +1312,10 @@ describe('Omnichannel Chat SDK', () => {
             }
         });
 
-        it('[LiveChatV1] ChatSDK.startChat() with customContext, browser, os, locale, device defined in sessionInitOptionalParams should pass it to OCClient.sessionInit() call\'s optional parameters', async() => {
```

Comment:
This test is valid. Its goal is still validate the `customContext` are being sent to `sessioninit`. We need to remove the `v1` dependency on it